[NEW] survey_record_generation (#149)
OCA/survey: survey_record_generation/readme/CONFIGURE.rst

Record generation configuration
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. |Image of record creation list| image:: ../static/description/record-creations.png

#. Go to the the survey

> #. In *Record creation* tab add a new line        

Reviewer: ```suggestion
#. In the *Record creation* tab, add a new line        
```

> #. Set a name for created record, select the Model of the record (eg: Prospect)

Reviewer: It is not clear reading this what the name will be used for. It also seems there are two steps. Or setting a name and selecting the model is the same thing?

#. Add a field configuration. So for each field : 

    .. |Image of record creation fields| image:: ../static/description/record-creation-customer.png

    #. You can check "unicity constraint" to prevent duplicates. 

>         In case of duplicates and if other record use this record to fill a m2o field, the founded record will be used

Reviewer: ```suggestion
        In case of duplicates and if other records use this record to fill a m2o field, the already present record will be used
```

>     #. You can configure explicitly where Odoo should retrieve the value of field : 

Reviewer: ```suggestion
    #. You can configure explicitly how Odoo should retrieve the value of field : 
```

>         * **fixed**: To set explicit value

Reviewer: ```suggestion
        * **fixed**: To set the value explicitly
```

>         * **question**: If value come from user's answer

Reviewer: ```suggestion
        * **question**: If the value comes from the user's answer
```

>             For m2o or m2m links, question should be configured before. See Question answers configuration section below.

Reviewer: ```suggestion
            For m2o or m2m links, the question should be configured before. See *Question answers configuration* section below.
```

>         * **other created record**: If value come from other created record (m2o case only)

Reviewer: ```suggestion
        * **other created record**: If the value comes from other created record (m2o case only)
```

Question answers configuration 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In a survey question : to configure value of choices

Reviewer: ```suggestion
In a survey question: to configure the value of choices
```

.. |Question answers| image:: ../static/description/question-answers.png

#. Configure a multiple choice question

> #. Select value type associated to answer:

Reviewer: ```suggestion
#. Select the value type associated with the answer:
```

    * **Value** > eg: to fill a selection field

>     * **Record** > to fill m2o or m2m field

Reviewer: ```suggestion
    * **Record** > to fill an m2o or m2m field
```

#. In case of *record* type: 

>     #. Select referenced model

Reviewer: ```suggestion
    #. Select the referenced model
```
